### PR TITLE
[WIP] Update content for hub page

### PR DESCRIPTION
**Pull request by richlander**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -19,22 +19,22 @@ ms.assetid:
 <article id="main">
     <section id="hero-content" class="graph">
         <h1>.NET Documentation</h1>
-        <h2>.NET is a development platform for any kind of app or workload, providing key capabilities for building high quality apps quickly. It consists of the common language runtime (CLR) and .NET libraries, which includes APIs that support an extensive range of technologies. .NET provides a managed execution environment, simplified development and deployment, and integration with a variety of programming languages, including C#, F# and Visual Basic.</h2>
+        <h2>.NET is a general purpose development platform that you can use for your next [cloud | desktop | mobile | web | IoT] app or game. It includes the C#, F# and Visual Basic programming languages, an extensive class library and API bindings for popular OSes. Start learning .NET today.</h2>
```
Comment (mairaw) [inline index.md]: nits:
OSes -> operating systems? I think it reads better, especially on a homepage
Should you end the paragraph with an exclamation point?

File: index.md [hunk 2]
```diff
@@ -75,25 +81,25 @@ ms.assetid:
...
                         </p>
                     </header>
                     <section class="journey-step-elements content">
                         <ul class="row">
                             <li class="column column-third">
-                                <a href="">
+                                <a href="https://docs.asp.net">
                                     <h3>ASP.NET Core</h3>
-                                    <p>Learn how to build web sites and applications using ASP.NET.</p>
+                                    <p>Learn how to build web sites and servicesb using ASP.NET Core.</p>
```
Comment (mairaw) [inline index.md]: servicesb -> services

**Comment by bradygaster** [comment]
@mairaw @richlander i fixed the typo. let me know if i'm okay to merge so i can take some screen shots.

**Comment by mairaw** [comment]
Sure, I'll merge now.

**Pull request merged**